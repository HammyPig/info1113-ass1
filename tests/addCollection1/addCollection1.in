ADD COLLECTION filename
add collection filename
aDd colLectiON filename