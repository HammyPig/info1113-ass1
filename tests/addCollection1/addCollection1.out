user: No such collection.

user: No such collection.

user: No such collection.

user: Ending Library process.
